data/brands/leisure/fitness_centre.json

```diff
@@ -1109,6 +1109,19 @@
         "name:en": "Genghis Khan Fitness Club",
         "name:zh": "成吉思汗健身俱樂部"
       }
+    },
+    {
+      "displayName": "Pure Fitness",
+      "locationSet": {"include": ["cn", "sg"]},
+      "tags": {
+        "brand": "Pure Fitness",
+        "brand:en": "Pure Fitness",
+        "brand:wikidata": "Q10852204",
+        "brand:wikipedia": "zh:Pure_Fitness",
```

Comment:
https://github.com/osmlab/name-suggestion-index/issues/6481

@tommylung wikipedia tag was removed from NSI project, I'll remove it, sorry （> <)